### PR TITLE
chore: add dedicated files for constants

### PR DESCRIPTION
**Pull request by eglitise**
This PR is a refactor with the following changes:
* Add a new `constants` directory for storing various constants used throughout the app
* Get rid of the `shared.js` and `AntdTypes.js` files whose purpose was not obvious
* Save the `automationName` value in lowercase, avoiding the need to change it to lowercase in other files

### CODE REVIEW

**Review by jlipps — APPROVED** [review]
seems reasonable to me!

File: app/renderer/components/ErrorBoundary/ErrorMessage.js [hunk 1]
```diff
@@ -2,13 +2,12 @@ import {CopyOutlined} from '@ant-design/icons';
 import {Alert, Button, Tooltip} from 'antd';
 import React from 'react';
 
+import {ALERT} from '../../constants/ANTD_TYPES';
+import {LINKS} from '../../constants/COMMON';
```
Comment (mykola-mokhnach) [inline app/renderer/components/ErrorBoundary/ErrorMessage.js:6]: Not sure we need to have uppercase file names. Is it mentioned in any standard?

File: app/renderer/utils/other.js [hunk 2]
```diff
@@ -28,3 +28,36 @@ export function addVendorPrefixes(caps) {
...
+}
+
+export function percentageToPixels(pct, maxPixels) {
+  if (!isNaN(pct)) {
+    return Math.round(maxPixels * (pct / 100));
+  }
+}
+
+export function parseCoordinates(element) {
+  let {bounds, x, y, width, height} = element.attributes || {};
```
Comment (mykola-mokhnach) [inline app/renderer/utils/other.js:45]: why cannot we use const?

Comment (eglitise) [reply app/renderer/utils/other.js:45]: I didn't modify these methods, but some variables can indeed be converted to consts. Not these though - only `bounds` is not changed. We can however just create new const variables, allowing to change these to consts too.

File: app/renderer/components/ErrorBoundary/ErrorMessage.js (hunk 1 above)
Comment (eglitise) [reply app/renderer/components/ErrorBoundary/ErrorMessage.js:6]: I saw this as a suggestion when searching for best practices for constants, and I thought it made sense. But it's probably not specified in any standard, no.

Comment (mykola-mokhnach) [reply app/renderer/components/ErrorBoundary/ErrorMessage.js:6]: lets only use lowercase for file names. It is already clear these are constants as they are located in the `constants` subdir

**Review by KazuCocoa — APPROVED** [review]
lg

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**